src/xunit.performance.core/Benchmark.cs

```diff
@@ -24,6 +24,11 @@ public static class Benchmark
         public static IEnumerable<BenchmarkIteration> Iterations => BenchmarkIterator.Current.Iterations;
 
         /// <summary>
+        /// Gets the inner iteration count (the count of actual iterations that should occur for each benchmark iteration).
+        /// </summary>
+        public static long InnerInterationCount => BenchmarkIterator.Current.InnerIterationCount;
+
+        /// <summary>
```

Comment:
_Sorry, I can't put the comment on the correct line because that line was not modified in your original change_

I think you also need to modify Iterate and IterateAsync to take care of inner iteration too.
i.e.
```C#
using (var measurement = iteration.StartMeasurement())
{
    for (long innerIteration = 0; innerIteration < InnerIterationCount; innerIteration++)
    {
        measuredOperation.Invoke();
    }
}
```
Similarly for IterateAsync